bin/run_tests

```diff
@@ -1,6 +1,17 @@
 #!/usr/bin/env bash
 set -e
 
+# Install bats for testing the bash scripts
```

Comment:
Don't think we should be running the install here every single time the script is run. Not only does it mean overriding a local install each run, but it isn't particularly configurable to install it in custom locations.

Instead, we should just check to see if bats is present on the runner's path and proceed or fail appropriately.